ios/Classes/InstabugFlutterPlugin.m

```diff
@@ -220,6 +220,13 @@ + (void) removeUserAttributeForKey:(NSString*) key {
     [Instabug removeUserAttributeForKey:key];
 }
 
+/**
+ * invoke sdk manually
+ */
++ (void) show {
+    [Instabug show];
```

Comment:
Do we need to make sure this call also happens on the main thread?

Reply:
I don't think so, it worked fine with me and its implemented similarly in RN